.github/workflows/pull-request.yml

```diff
@@ -39,8 +39,7 @@ jobs:
           base_sha: ${{ steps.main_sha.outputs.main_sha }}
        
       - name: Create Pull Request
-        if: (steps.check_readme_exists.outputs.files_exists == 'true' && contains(steps.changed-files.outputs.all_changed_files, 'README.md'))
-          || (steps.check_changelog_exists.outputs.files_exists == 'true' && contains(steps.changed-files.outputs.all_changed_files, 'CHANGELOG.md'))
+        if: steps.check_changelog_exists.outputs.files_exists == 'true' && contains(steps.changed-files.outputs.all_changed_files, 'CHANGELOG.md')
```

Comment:
@dpappalardoMedallia Your Android version looks incorrect, please update it to match Joe's version.